src/modules/transfer/components/MosaicInput.vue

```diff
@@ -1,16 +1,44 @@
 <template>
   <div>
+    <!-- showing blue text of select assets -->
     <div class="h-5 text-left">
       <transition enter-active-class="animate__animated animate__fadeInUp">
-        <span v-if="showSelectTitle" class="text-xs text-blue-400 ">{{ placeholder }}</span>
+        <span model-v="showSelectTitle" class="text-xs text-blue-400 ">{{ placeholder }}</span>
```

Comment:
I think it is better to have the Select Asset to be there all the time. Instead of just appearing after an Asset is selected
![Screenshot 2022-12-30 144640](https://user-images.githubusercontent.com/59526011/210042456-75f1fd9b-ede7-468a-b05e-14b3db6eef18.png)
I think I will delete the `showSelectTitle` variable.

Reply:
ok